distributor-api-v1/operations.md

```diff
@@ -312,7 +312,7 @@ Alternative initial call used to obtain all static data about hotel relevant for
             "Name": "Czech Republic"
         }
     ],
-    "ImageBaseUrl": "https://cdn.demo.mews.li/Media/Image",
+    "ImageBaseUrl": "https://app.mews-demo.com/Media/Image",
```

Comment:
Are you sure about this? Why do we want to bypass CDN suddenly? CC @skurik

Reply:
Good catch. But then we should have `cdn.mews-demo.com` I guess.

Reply:
I did call the API on `develop` and the URL looked like this. But agree it should be `cdn.mews-demo.com`

@skurik let me know once we will have it again and I will change it in the docs

Reply:
https://github.com/MewsSystems/gitbook-distributor-guide/pull/45

Reply:
CDN setup under progress, waiting for the HTTPS certificate to be deployed. Will let you know when it's ready.

Reply:
Thx, so I will not revert it for now and then just merge the #46 👍

Reply:
Done.

![image](https://user-images.githubusercontent.com/951352/111862914-94b5c580-8958-11eb-913b-2306aeba51da.png)